source/mir/math/stat.d

```diff
@@ -1626,6 +1627,704 @@ unittest
     assert(x.center!(mean!"precise").all!approxEqual(result));
 }
 
+struct MapSummator(alias fun, T, Summation summation) 
+    if(isMutable!T)
+{
+    ///
+    Summator!(T, summation) mapSummator;
```

Comment:
The inner summator, while the struct itself is mapSummator. Plus there is no other summators here. Please cal it just `summator`